### PR TITLE
feat(SAMS-36): Individual Module Page

### PR DESCRIPTION
**Pull request by vogelino**

### CODE REVIEW

File: src/modules/06.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
+module_number: '06'
```
Comment (dnsos) [inline src/modules/06.md:2]: I _believe_ we could also get the file title from [Eleventy supplied data](https://www.11ty.dev/docs/collections/#collection-item-data-structure). I also haven't done that in the overview page, but perhaps we could refactor this in the future. Consider this comment optional.

**Review by dnsos — CHANGES_REQUESTED** [review]
Hey @vogelino and @Lisa-Stubert, great work on this page!

I have only a few comments that should be easy to address. Looking at my changes on the modules overview, I'm noticing that we could unify some stuff, specifically I could make use of some your partials, I think. But that's a task for after this PR.

File: src/modules/06.md (hunk 1 above)
Comment (vogelino) [reply src/modules/06.md:2]: I found out a way to use Eleventy supplied data in the overview, as demonstrated in the linked documentation page. Unfortunately however, this doesn't work within individual layout pages. At least, I am not able to find out how. Do you have an idea?

Comment (vogelino) [reply src/modules/06.md:2]: Forger it! I found a way. You can access that on the `page` object. E.g. `{{ page.fileSlug }}`

**Review by dnsos — APPROVED** [review]
Great changes!

Just one more thing before merging: Can you make sure that the `<ol>`'s, `<ul>`'s and `<strong>`'s within prose use the blue color instead ob black/gray?

Screenshot 2022-03-17 at 11 12 12

Screenshot 2022-03-17 at 11 11 55

**Comment by vogelino** [comment]
> Can you make sure that the <ol>'s, <ul>'s and <strong>'s within prose use the blue color instead ob black/gray?

I configured the typography plugin to use our colors! I should work better now! :)

File: tailwind.config.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ module.exports = {
         500: "#393a60",
         600: "#2e2e4d",
         700: "#22233a",
-        800: "#171726",
+        500: "#171726",
```
Comment (dnsos) [inline tailwind.config.js:35]: Was this change intentional?

Comment (vogelino) [reply tailwind.config.js:35]: Oh no! Good catch! I will revert this!

Comment (vogelino) [reply tailwind.config.js:35]: @dnsos we never use the gray colors in this website. Should we keep them in the config?

Comment (dnsos) [reply tailwind.config.js]: Good point. Feel free to remove them with this PR or separately.

**Review by dnsos — APPROVED** [review]
